### PR TITLE
refactor: Use keyboard shortcut hook in article form

### PR DESCRIPTION
**Pull request by Link2Twenty**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Now we have a helper for keyboard shortcuts we can move old shortcuts over to this new method

## Related Tickets & Documents
#10878

## QA Instructions, Screenshots, Recordings
Press ctrl/command + shift + p in the new post page
it should toggle the preview

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.forem.com
- [ ] readme
- [x] no documentation needed

## [optional] Are there any post deployment tasks we need to perform?
N/A

## [optional] What gif best describes this PR or how it makes you feel?

Mad laughter

### CODE REVIEW

**Comment by reobin** [comment]
Much cleaner now 🌞

Are you planning on doing all of the ones mentioned in #10878 in this PR?

**Comment by Link2Twenty** [comment]
I think there's just the modal closing left?
Feel free to take that one if you want it 🙂

**Comment by reobin** [comment]
You're right, I thought there were more. I'll take the last one 👍

File: app/javascript/article-form/articleForm.jsx [hunk 1]
```diff
@@ -361,7 +361,9 @@ export default class ArticleForm extends Component {
           submitting={submitting}
         />
 
-        <KeyboardShortcutsHandler togglePreview={this.fetchPreview} />
+        <KeyboardShortcuts shortcuts={{
+          "ctrl+shift+KeyP": e => this.fetchPreview(e)
```
Comment (reobin) [inline app/javascript/article-form/articleForm.jsx:365]: I think by prettier's rules in this repo, this line needs `(e)` and a trailing comma

**Review by rhymes — APPROVED** [review]
Works well, just a note: the instructions say `ctrl alt p` not `ctrl shift p`

**Review by benhalpern — APPROVED** [review]

**Pull request merged**